lib/db/statement.js

```diff
@@ -568,6 +581,20 @@ Statement.prototype.buildStatement = function () {
     }, this).filter(function (item) { return item; }).join(', '));
   }
 
+  if (state.put && state.put.length) {
+    statement.push('PUT');
+    statement.push(state.put.map(function (item) {
+      var objectToAdd = item[1];
+      var keys = Object.keys(objectToAdd);
+      var propertyName = item[0];
+      return keys.map(function (key) {
+        var paramName = 'param' + paramify(propertyName + key) + (this._state.paramIndex++);
+        this.addParam(paramName, objectToAdd[key]);
+        return propertyName + ' = \'' + key + '\', :' + paramName;
```

Comment:
let's use double quotes (not single) around the key name and also use `utils.escape()` on it, just in case.